com.stansassets.build/Editor/Steps/UnityPlayerBuildStep.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace StansAssets.Build.Editor
+{
+    class UnityPlayerBuildStep : IBuildStep
+    {
+        private int m_Priority;
+
+        public int Priority => m_Priority;
+
+        public bool Execute(BuildContext buildContext)
+        {
+            return BuildProject();
+        }
+        
+        private bool BuildProject()
+        {
+            bool isSuccess = true;
+
+            BuildPlayerOptions defaultBuildPlayerOptions = new BuildPlayerOptions();
+            BuildPlayerOptions currentBuildPlayerOptions =
+                BuildPlayerWindow.DefaultBuildMethods.GetBuildPlayerOptions(defaultBuildPlayerOptions);
```

Comment:
I think we need to give an ability to define build option to the person who starts the build.
By default, we have options provided when the Build button is clicked in the build Player settings window